packages/hooks/src/configure.ts

```diff
@@ -0,0 +1,15 @@
+export type ErrorNotifierFunction = (message: string, error: unknown) => void;
+
+export interface ConfigurationOptions {
```

Comment:
Does this need to be exported?

Reply:
If it's an input type I like it being exported mostly for the sake of people that maybe implement it on a previous line rather than inline.

```
  const myOptions: ConfigurationOptions = {
    //...
  };
  
  Configuration.configure(myOptions);
```

Reply:
If the type was only used internally I wouldn't export it.